x-pack/plugins/cloud_security_posture/public/components/column_name_with_tooltip.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiToolTip, EuiToolTipProps } from '@elastic/eui';
+import React, { ReactNode } from 'react';
+
+export const ColumnNameWithTooltip = ({
+  tooltipContent,
+  columnName,
+}: {
+  tooltipContent: EuiToolTipProps['content'];
+  columnName: ReactNode;
+}) => (
+  <EuiToolTip content={tooltipContent}>
+    <EuiFlexGroup gutterSize="xs" alignItems="center">
+      <EuiFlexItem>
+        <span>{columnName}</span>
```

Comment:
we might need to add the `truncate` class name here?

Reply:
good point but from just using this class on the title span it didn't add the ellipsis. this is something we need to figure out in general but not worth wasting time on right now